src/components/BaseCard.vue

```diff
@@ -1,61 +1,59 @@
 <template>
-  <div
-    class="flex flex-col overflow-hidden rounded-lg shadow-md hover:shadow-xl transition-shadow"
+  <router-link
+      v-if="+get(post, 'relationships.creator.id', null) === +userState.id"
```

Comment:
Let's extract this to a computed property or a method instead.
Also would suggest `.toString()` for the properties instead of casting to numbers with `+`